tests/framework/population/test_manager.py

```diff
@@ -66,6 +66,13 @@ def test_initializer_set_duplicate_columns():
         component_set.add(component2.initializer, ["sneaky_column"] + columns)
 
 
+def test_initializer_set_population_manager():
+    component_set = InitializerComponentSet()
+    population_manager = PopulationManager()
+
+    component_set.add(population_manager.on_initialize_simulants, ["tracked"])
+
+
```

Comment:
This is just testing you can add a component to component_set?

Reply:
This is testing that the PopulationManager can have an initializer. Currently this test isn't important, but in the PR for https://jira.ihme.washington.edu/browse/MIC-4455 this will be important.